python/astshim/test.py

```diff
@@ -131,3 +132,66 @@ def checkBasicSimplify(self, amap):
             self.assertEqual(mb.getNout(), cmp3.getNout())
             self.assertEqual(cmp3.getNin(), cmp3simp.getNin())
             self.assertEqual(cmp3.getNout(), cmp3simp.getNout())
+
+
+def makeCoeffs(nIn, nOut):
+    """Make an array of coefficients for astshim.PolyMap for the following equation:
+
+    fj(x) = C0j x0^2 + C1j x1^2 + C2j x2^2...
+    where:
+    * i ranges from 0 to nIn-1
```

Comment:
That's not the correct final term; it's `C(nIn-1)j x(nIn-1)^2`. Maybe that's why you left it out in the original description?

While we're updating this function, `Cij` is actually coded as `0.001 (i + j + 1)` both here and in afw. Should I change the documentation to match the code, or the code to match the documentation?